.github/workflows/verify_dist.yml

```diff
@@ -0,0 +1,35 @@
+name: Verify distribution package
+permissions:
+  contents: read
+
+on:
+  push:
+    branches:
+      - develop
+  pull_request:
+    branches:
+      - develop
```

Comment:
I would only run this check on push for two reasons. First, I don’t want to make every collaborator wait an extra minute, especially since most mistakes occur in documentation or configuration files rather than in the code itself. Second, GitHub imposes a limit on the number of checks we can run, so adding this would bring us closer to that limit while providing minimal benefit.